test/test.js

```diff
@@ -4,44 +4,11 @@ var url = require('url');
 var expect = require('chai').expect;
 var objectAssign = require('object-assign');
 var format = require('string-format');
-var rimraf = require("rimraf");
-var fs = require('fs');
 var common = require('../lib/common')
 
-/*rimraf('sitespeed-result', function(err) {
-    console.log("delete legacy test data success")
-});
-*/
-
-var path1 = __dirname + '/../../../config/budget-testcases.json';
-var path2 = __dirname + '/../config/budget-testcases.json';
-var config = process.env.BUDGET_TESTCASES;
-if (!config) {
-    if (fs.existsSync(path1)) {
```

Comment:
@kakazuo Ok, sure... I can see that argument... maybe we should at least provide a little code snippet in the README that shows an example of loading that file and calling `performBudgetTest`